cvat/apps/documentation/user_guide.md

```diff
@@ -1,232 +1,446 @@
+    - [Task synchronization with a repository](#task-synchronization-with-a-repository)
+    - [Vocabulary](#vocabulary)
+    - [Workspace — context menu](#workspace--context-menu)
+    - [Settings](#settings)
+    - [Bottom panel](#bottom-panel)
+    - [Side panel](#side-panel)
+      - [Objects](#objects)
+      - [Labels](#labels)
+    - [Bottom side panel](#bottom-side-panel)
+    - [Fullscreen player mode](#fullscreen-player-mode)
+  - [Annotation mode (advanced)](#annotation-mode-advanced)
+  - [Interpolation mode (advanced)](#interpolation-mode-advanced)
+  - [Attribute annotation mode (advanced)](#attribute-annotation-mode-advanced)
+  - [Annotation with polygons](#annotation-with-polygons)
+  - [Annotation with polylines](#annotation-with-polylines)
+  - [Annotation with points](#annotation-with-points)
+  - [Annotation with Auto Segmentation](#annotation-with-auto-segmentation)
+  - [Shape grouping](#shape-grouping)
+  - [Filter](#filter)
+  - [Analytics](#analytics)
+  - [Shortcuts](#shortcuts)
+  - [Hints](#hints)
+
 # User's guide
 
-Computer Vision Annotation Tool (CVAT) is a web-based tool which helps to annotate video and images for Computer Vision algorithms. It was inspired by [Vatic](http://carlvondrick.com/vatic/) free, online, interactive video annotation tool. CVAT has many powerful features: __interpolation of bounding boxes between key frames, automatic annotation using TensorFlow OD API, shortcuts for most of critical actions, dashboard with a list of annotation tasks, LDAP and basic authorization, etc...__ It was created for and used by a professional data annotation team. UX and UI were optimized especially for computer vision tasks developed by our team.
+Computer Vision Annotation Tool (CVAT) is a web-based tool which helps to
+annotate videos and images for Computer Vision algorithms. It was inspired
+by [Vatic](http://carlvondrick.com/vatic/) free, online, interactive video
+annotation tool. CVAT has many powerful features: _interpolation of bounding
+boxes between key frames, automatic annotation using deep learning models,
+shortcuts for most of critical actions, dashboard with a list of annotation
+tasks, LDAP and basic authorization, etc..._ It was created for and used by
+a professional data annotation team. UX and UI were optimized especially for
+computer vision tasks developed by our team.
 
 ## Getting started
 
 ### Authorization
-- First of all you have to log in to CVAT tool.
+-   First of all, you have to log in to CVAT tool.
 
     ![](static/documentation/images/image001.jpg)
 
     ![](static/documentation/images/image002.jpg)
 
-- If you don't have an account you have to create it using the link below the login page.
+-   If you don't have an account, you have to create it using the link below on the login page.
 
     ![](static/documentation/images/image003.jpg)
 
 ### Administration panel
 Type ``/admin`` in URL to go to the administration panel.
```

Comment:
Go to [Django administration panel](http://localhost:8080/admin).

Note: need to insert a screenshot here for permissions (go to admin/auth/user/44/change and create a screenshot of permissions section)